### PR TITLE
Allow PPO to turn off advantage normalization

### PR DESCRIPTION
**Pull request by vwxyzjn**
## Description
Allow PPO to turn of advantage normalization. Follow up from https://github.com/DLR-RM/stable-baselines3/pull/763, #60

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/blob/master/CONTRIBUTING.md) guide (**required**)
- [ ] The functionality/performance matches that of the source (**required** for new training algorithms or training-related features).
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have included an example of using the feature (*required for new features*).
- [ ] I have included baseline results (**required** for new training algorithms or training-related features).
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the changelog accordingly (**required**).
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: tests/test_run.py [hunk 1]
```diff
@@ -142,3 +142,9 @@ def make_env():
         train_freq=5,
     )
     model.learn(total_timesteps=150)
+
+
+@pytest.mark.parametrize("normalize_advantage", [False, True])
+def test_advantage_normalization(model_class, normalize_advantage):
+    model = MaskablePPO("MlpPolicy", "CartPole-v1", n_steps=64, normalize_advantage=normalize_advantage)
```
Comment (araffin) [inline tests/test_run.py:149]: not sure if it works with CartPole, let's see...

**Review by araffin — COMMENTED** [review]
Please update the changelog too, otherwise LGTM ;) (and you need an additional issue if you have already one opened in SB3 repo)

File: tests/test_run.py (hunk 1 above)
Comment (vwxyzjn) [reply tests/test_run.py:149]: Hey you may need to approve the workflow run ;)

Comment (araffin) [reply tests/test_run.py:149]: i did and there is some failures already ;)

**Comment by vwxyzjn** [comment]
Had the following error, seems to relate to https://github.com/DLR-RM/stable-baselines3/pull/782

```
ImportError: cannot import name 'GoalEnv' from 'gym' (/home/costa/.cache/pypoetry/virtualenvs/cleanrl-ghSZGHE3-py3.9/lib/python3.9/site-packages/gym/__init__.py)
```

**Comment by vwxyzjn** [comment]
Ok should be good now.

**Review by araffin — APPROVED** [review]

**Pull request merged**